src/containers/MainContainer/index.jsx

```diff
@@ -133,10 +133,24 @@ class MainContainer extends Component {
       event.preventDefault();
       const product = event.target.parentElement.getAttribute('product');
       const component = event.target.parentElement.getAttribute('component');
-      this.setState(prevState => ({
-        showComponent: prevState.bugzillaComponents[`${product}::${component}`],
-        showPerson: undefined,
-      }));
+      const teamKey = event.target.parentElement.getAttribute('teamkey');
+      if (teamKey) {
+        this.setState(prevState => ({
+          showComponent: {
+            title: prevState.teamComponents[teamKey].label,
+            ...prevState.teamComponents[teamKey],
+          },
+          showPerson: undefined,
```

Comment:
`showPerson` sounds like a boolean but is set to `undefined` in both conditions. Could you clarify this?